mars_api/models.py

```diff
@@ -1,8 +1,109 @@
 from django.contrib.auth.models import User
 from django.db import models
+from django.utils import timezone
+from model_utils import Choices
+
+CORPORATIONS = Choices(
+    "Aphrodite",
+    "Aridor",
+    "Arklight",
+    "Celestic",
+    "Cheung Shing Mars",
+    "Credicor",
+    "Ecoline",
+    "Helion",
+    "Interplanetary Cinematics",
+    "Inventrix",
+    "Manutech",
+    "Mining Guild",
+    "Morning Star Inc.",
+    "Phobolog",
+    "Point Luna",
+    "Polyphemos",
+    "Poseidon",
+    "Robinson Industries",
+    "Saturn Systems",
+    "Storm Craft Incorporated",
+    "Terractor",
+    "Tharsis Republic",
+    "Thorgate",
+    "United Nations Mars Initiative",
+    "Valley Trust",
+    "Viron",
+    "Vitor",
+)
+
+
+MAPS = Choices("Tharsis", "Elysium", "Hellas")
 
 
 class Player(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE, null=True, blank=True)
     nickname = models.CharField(max_length=32, unique=True)
     motto = models.CharField(max_length=100, blank=True, null=True)
+
+    def __str__(self):
+        return f"{self.nickname}"
+
+
+class Game(models.Model):
+    date = models.DateTimeField(default=timezone.now)
+    game_map = models.CharField(choices=MAPS, default=MAPS.Tharsis, max_length=14)
+
+    draft_variant = models.BooleanField(default=True)
+    prelude = models.BooleanField(default=False)
+    venus_next = models.BooleanField(default=False)
+    colonies = models.BooleanField(default=False)
+
+    def __str__(self):
+        return f"id: {self.id}; date: {self.date}; game_map: {self.game_map}"
+
+    class Meta:
+        constraints = [
+            models.CheckConstraint(
+                check=models.Q(game_map__in=[map_[0] for map_ in dict(MAPS).items()]),
+                name=f"Only {[map_[0] for map_ in dict(MAPS).items()]} maps are allowed.",
```

Comment:
Thought about writing a subclass for `MAPS` and `CORPORATIONS` but I only need list format in these cases, so left it like that.

Reply:
You can do `list(dict(MAPS).values())` here (or keys()).

Reply:
Or even better - `set`

Reply:
`list(set(dict(MAPS)))` ? :D